addons/binding/org.openhab.binding.heos/ESH-INF/thing/Channels.xml

```diff
@@ -0,0 +1,104 @@
+     <channel-type id="group">
+        <item-type>Switch</item-type>
+        <label>Connect</label>
+     </channel-type>
+     <channel-type id="ungroup">
+        <item-type>Switch</item-type>
+        <label>Group</label>
+     </channel-type>
+     <channel-type id="title">
+        <item-type>String </item-type>
+        <label>Song</label>
+     </channel-type>
+     <channel-type id="interpret">
+        <item-type>String</item-type>
+        <label>Artist</label>
+     </channel-type>
+     <channel-type id="album">
+        <item-type>String</item-type>
+        <label>Album</label>
+     </channel-type>
+     <channel-type id="imageUrl" advanced="true">
+        <item-type>String</item-type>
+        <label>Cover</label>
+     </channel-type>
+     <channel-type id="player">
+        <item-type>Switch</item-type>
+        <label>Player</label>
+     </channel-type>
+     <channel-type id="buildGroup">
+        <item-type>Switch</item-type>
+        <label>Made Group</label>
+     </channel-type>
+     <channel-type id="dynamicGroupHandling">
+        <item-type>Switch</item-type>
+        <label>Keep Groups</label>
+     </channel-type>
+     <channel-type id="favorit">
+        <item-type>Switch</item-type>
+        <label>Favorites</label>
+     </channel-type>
+     <channel-type id="playlists" advanced="true">
+        <item-type>String</item-type>
+        <label>Playlists</label>
+     </channel-type>
+     <channel-type id="reboot">
+        <item-type>Switch</item-type>
+        <label>Reboot</label>
+     </channel-type>
+     <channel-type id="input" advanced="true">
+        <item-type>String</item-type>
+        <label>Input Source</label>
+     </channel-type>
+     <channel-type id="currentPosition" advanced="true">
+        <item-type>String</item-type>
+        <label>Current track position</label>
+     </channel-type>
+     <channel-type id="duration" advanced="true">
+        <item-type>String</item-type>
+        <label>Track duration</label>
+     </channel-type>
+     <channel-type id="rawCommand" advanced="true">
+        <item-type>String</item-type>
+        <label>Sending a HEOS command as specified within the HEOS CLI protocol</label>
+     </channel-type>
+     <channel-type id="type" advanced="true">
+        <item-type>String</item-type>
+        <label>The media currently played type (station, song, ...)</label>
+     </channel-type>
+     <channel-type id="station" advanced="true">
+        <item-type>String</item-type>
+        <label>The station name</label>
+     </channel-type>
+     <channel-type id="playUrl" advanced="true">
+        <item-type>String</item-type>
+        <label>Play a media file from URL</label>
+     </channel-type> 
+     <channel-type id="shuffleMode" advanced="true">
+        <item-type>Switch</item-type>
+        <label>Sets the shuffle mode</label>
+     </channel-type>
+     <channel-type id="repeatMode" advanced="true">
+        <item-type>String</item-type>
```

Comment:
Should this also be a switch? Or at least the same as for shuffleMode

Reply:
The repeat mode can have 3 states: "One" "All" and "Off". That's why I used a String.